sound/soc/sof/ipc4-topology.c

```diff
@@ -1888,6 +1932,7 @@ static int sof_ipc4_prepare_process_module(struct snd_sof_widget *swidget,
 	if (ret < 0)
 		return ret;
 
+	/* No need to check the return value. Some processing modules do not have output pins */
 	ret = sof_ipc4_init_output_audio_fmt(sdev, &process->base_config, available_fmt, ret);
```

Comment:
Why don't we just call `sof_ipc4_init_output_audio_fmt(sdev, &process->base_config, available_fmt, ret);` if we don't check the return value?

Reply:
because it is used just a few lines below

Reply:
This looks a bit odd in the series. This gets used in a later patch in the series, but makes little sense in this one as far as I can tell.

Reply:
@kv2019i no "ret" is used on line 1940 even in this patch like it was before this patch a well